core/tests/webdriverio_utils/AdminPage.js

```diff
@@ -0,0 +1,126 @@
+// Copyright 2022 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Page object for the admin page, for use in WebdriverIO
+ * tests.
+ */
+
+var action = require('./action.js');
+var forms = require('./forms.js');
+var general = require('./general.js');
+var waitFor = require('./waitFor.js');
+
+var AdminPage = function() {
+  var ADMIN_URL_SUFFIX = '/admin';
+  var addNewRoleButton = $('.e2e-test-add-new-role-button');
+  var adminRolesTab = $('.e2e-test-admin-roles-tab');
+  var adminRolesTabContainer = $('.e2e-test-roles-tab-container');
+  var configTab = $('.e2e-test-admin-config-tab');
+  var editUserRoleButton = $('.e2e-test-role-edit-button');
+  var progressSpinner = $('.e2e-test-progress-spinner');
+  var roleEditorContainer = $('.e2e-test-roles-editor-card-container');
+  var roleSelector = $('.e2e-test-new-role-selector');
+  var saveAllConfigs = $('.e2e-test-save-all-configs');
+  var statusMessage = $('.e2e-test-status-message');
+  var usernameInputFieldForRolesEditing = $(
+    '.e2e-test-username-for-role-editor');
+
+  var saveConfigProperty = async function(
+      configProperty, propertyName, objectType, editingInstructions) {
+    await waitFor.visibilityOf(
+      configProperty.$('.e2e-test-config-title'),
+      'Config Title taking too long too appear');
+    var title = await configProperty.$('.e2e-test-config-title').getText();
+    if (title.match(propertyName)) {
+      await editingInstructions(
+        await forms.getEditor(objectType)(configProperty));
+      await action.click('Save All Configs Button', saveAllConfigs);
+      await general.acceptAlert();
+      // Waiting for success message.
+      await waitFor.textToBePresentInElement(
+        statusMessage, 'saved successfully',
+        'New config could not be saved');
+      return true;
+    }
+  };
+
+  this.get = async function() {
+    await browser.url(ADMIN_URL_SUFFIX);
+    await waitFor.pageToFullyLoad();
+  };
+
+  var _switchToRolesTab = async function() {
+    await action.click('Admin roles tab button', adminRolesTab);
+
+    await expect(await adminRolesTab.getAttribute('class')).toMatch('active');
+    await waitFor.visibilityOf(
+      adminRolesTabContainer, 'Roles tab page is not visible.');
+  };
```

Comment:
Please place it at the top as done for saveConfigProperty.

Reply:
Done.